src/react-apps/applications/shared/src/version-control/versionControlHeader.tsx

```diff
@@ -0,0 +1,392 @@
+import { createMuiTheme, createStyles, Grid, WithStyles, withStyles } from '@material-ui/core';
+import * as React from 'react';
+import altinnTheme from '../theme/altinnStudioTheme';
+import { getLanguageFromKey } from '../utils/language';
+import FetchChangesComponent from '../version-control/fetchChanges';
+import ShareChangesComponent from '../version-control/shareChanges';
+import SyncModalComponent from './syncModal';
+
+export interface IVersionControlHeaderProps extends WithStyles<typeof styles> {
+  language: any;
+}
+
+export interface IVersionControlHeaderState {
+  changesInMaster: boolean;
+  changesInLocalRepo: boolean;
+  moreThanAnHourSinceLastPush: boolean;
+  hasPushRight: boolean;
+  anchorEl: any;
+  modalState: any;
+  mergeConflict: boolean;
+}
+
+const theme = createMuiTheme(altinnTheme);
+
+const styles = createStyles({
+  headerStyling: {
+    background: theme.altinnPalette.primary.greyLight,
+    paddingTop: 10,
+  },
+});
+
+const initialState = {
```

Comment:
Change from 'initialState' to 'initialModalState' or something?

Reply:
fixed